src/Client/Client.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
Shouldn't we use the client from Facebook instead?

Reply:
It will be tricky. As we don't store Facebook's Event objects, but normalize them and store normalized data at PixelEvent instead. So we should use our own Client, I believe.

Reply:
After live test and few fixes - our client seems like works well